### PR TITLE
Add admin codes

### PR DESCRIPTION
**Pull request by RadekMolenda**

### CODE REVIEW

File: lib/gemonames.rb [hunk 1]
```diff
@@ -66,11 +66,16 @@ def wrap_in_search_result(result)
...
+        admin_id3: result["adminId3".freeze],
+        admin_id2: result["adminId2".freeze],
+        admin_id1: result["adminId1".freeze],
+        country_id: result["countryId".freeze]
       )
     end
   end
 
-  SearchResult = Value.new(:geoname_id, :name, :country_code) do
+  SearchResult = Value.new(:geoname_id, :name, :country_code, :admin_id4, :admin_id3, :admin_id2, :admin_id1, :country_id) do
```
Comment (tomekw) [inline lib/gemonames.rb]: Break this line?

File: lib/gemonames.rb [hunk 2]
```diff
@@ -66,11 +66,25 @@ def wrap_in_search_result(result)
         geoname_id: result.fetch("geonameId".freeze),
         name: result.fetch("name".freeze),
         country_code: result.fetch("countryCode".freeze),
+        admin_id4: result["adminId4".freeze],
+        admin_id3: result["adminId3".freeze],
+        admin_id2: result["adminId2".freeze],
+        admin_id1: result["adminId1".freeze],
+        country_id: result["countryId".freeze]
```
Comment (mfilej) [inline lib/gemonames.rb]: Trailing comma please.

Comment (RadekMolenda) [inline lib/gemonames.rb]: I don't like it :cry:

**Comment by mfilej** [comment]
:sparkles:

**Pull request merged**